articles/05-FAQ.md

```diff
@@ -0,0 +1,96 @@
+# Mixed Reality Toolkit (XRTK) Frequently Asked Questions
+
+![The Mixed Reality Toolkit](/images/Branding/XRTK_Logo_1200x250.png)
+
+## 1. How do I create a new Mixed Reality Toolkit Project?
+
+We recommend walking through the [Getting Started guide](00-GettingStarted.md) for a detailed understanding to get up and running.
```

Comment:
Why duplicate text here instead of just providing links to the specific articles?

Reply:
As the FAQ explains, it's a short short version for those who don't want to follow the full guide.  One of the feedbacks from the recent "complexities" with the old project was it was too hard to find stuff and took to much reading to get started.

This isn't a replacement, just a quick "how to"